src/main/java/com/courtcanva/ccfranchise/exceptions/NoAvailableOrderException.java

```diff
@@ -0,0 +1,8 @@
+package com.courtcanva.ccfranchise.exceptions;
+
+public class NoAvailableOrderException extends RuntimeException {
```

Comment:
since you have defined a new exception, did you add to controlleradvisor to handle it?